yw_website/yw_website/apps/website/serializers.py

```diff
@@ -159,21 +127,26 @@ def update(self, validated_data):
             }
         )
         v.save()
+        r = self.__create_update_helper(w, v, validated_data)
+
+        return w.pk, v.pk, r.pk, new_version
+
+    def __create_update_helper(self, w, v, validated_data):
```

Comment:
could you split this into two methods so that it's more obvious whats happening? like `create_run(self, w, v)` and `update_workflow_tags(self, w, tags)`

Reply:
Yea sure